site/app/templates/forum/ThreadPostForm.twig

```diff
@@ -42,38 +47,38 @@
         </label>
         <script>
             $(document).ready(function () {
-                        flatpickr(".date_picker", {
-                            plugins: [ShortcutButtonsPlugin(
+                flatpickr(".date_picker", {
```

Comment:
Is this change required? Seems just like a spacing change.

Reply:
Was just making the code in proper alignment..